v3/src/components/web-view/web-view.tsx

```diff
@@ -26,6 +30,68 @@ export const WebViewComponent = observer(function WebViewComponent({ tile }: ITi
       <div className="codap-web-view-iframe-wrapper">
         <iframe className="codap-web-view-iframe" ref={iframeRef} src={webViewModel.url} />
       </div>
+      <WebViewDropOverlay />
     </div>
   )
 })
+
+// The WebViewDropOverlay broadcasts notifications to plugins as the user drags and drops attributes.
+function WebViewDropOverlay() {
```

Comment:
Even though it's not that large, I think it would be worth moving this into its own file so it's easier to find.